win32/glob.c

```diff
@@ -913,7 +872,7 @@ static int globextend(const Char *path, glob_t *pglob, struct glob_lim *limitp,
 				return(GLOB_NOSPACE);
 			}
 			if ((statv[pglob->gl_offs + pglob->gl_pathc] =
-				malloc(sizeof(**statv))) == NULL)
+				emalloc(sizeof(**statv))) == NULL)
```

Comment:
checking emalloc result is incorrect as it can never be NULL. If there's an error, it will bail out. So this should get changed